### PR TITLE
Create Order Page View.md

### PR DESCRIPTION
**Pull request by MatShir**
The new specs for the Order Project.

### CODE REVIEW

File: Order Page View.md [hunk 1]
```diff
@@ -0,0 +1,338 @@
...
+
+When the user clicks on the button, it downloads the invoice.
+
+4.  **View delivery slip**
+
+If the current or new order status as **the generation of the delivery slip** then **the button to download the delivery slip is shown.**
+
+5.  **Partial refund button**
+
+If the current or new order status of order considered **as paid** then **the button is shown.** After clicking on the button, then the product table will take the full width of the page. The user can edit the refund amount by product and also the shipping cost. He must select the quantity of the product concerned for the refunded and then write refund amount chosen by product rows. The maximum refundable amount is written bellow all the amount input in tax included.
```
Comment (Robin-Fischer-PS) [inline Order Page View.md]: There is a missing information : What about an order which has a current (or new) status "not paid" (awaiting for bank wire), but which was previously in a "paid" status ? Actually, in both Legacy and migrated code, the check is on the presence (or not) of an invoice

Comment (Robin-Fischer-PS) [inline Order Page View.md]: "The maximum refundable amount is written bellow all the amount input in tax included" : Should this maximal refundable amount be checked when validating the partial refund ? Or is this just "for information" for the merchant ? (In Legacy it was just "For information")

File: Order Page View.md [hunk 2]
```diff
@@ -0,0 +1,338 @@
...
+
+By default, the quantity input has 0 prefilled. To validate a partial refund, the user must must as selected minimun 1 quantity.
+
+**Before the validating a partial refund,** the user can choose to re-stock or generate a voucher by checking the checkboxes.
```
Comment (Robin-Fischer-PS) [inline Order Page View.md]: Need more info on "generate a voucher" or "generate a credit slip" choices

**Comment by MatShir** [comment]
https://github.com/PrestaShop/PrestaShop/issues/17032
Need to update the behaviors when deleting customer related to the order

**Comment by MatShir** [comment]
Need to update specs bc of #17648

**Comment by MatShir** [comment]
Need to update specs bc of PrestaShop/PrestaShop/pull/17852

**Comment by MatShir** [comment]
https://github.com/PrestaShop/PrestaShop/pull/17938
Paying with another currency, the change rate is based the latest conversion rate of the shop

**Comment by MatShir** [comment]
https://github.com/PrestaShop/PrestaShop/issues/17916
Date in RTL is in the occidental numeric system but the date selector has Arabic translation

**Comment by MatShir** [comment]
https://github.com/PrestaShop/PrestaShop/issues/17968
Can't add product with 0 quantity

**Comment by MatShir** [comment]
https://github.com/PrestaShop/PrestaShop/issues/17971
The stock is not correctly recalculated after editing product quantity in the order

**Comment by MatShir** [comment]
Editing an address in the order should create a new address PrestaShop/PrestaShop#17978
Address edition

File: Order Page View.md (hunk 1 above)
Comment (MatShir) [reply Order Page View.md]: Does it mean a generated an invoice change the status of the order as paid? Or it just follows if the order has an invoice ?
Thank you for your feedback

Comment (MatShir) [reply Order Page View.md]: Seen with the core dev, the button will appear when the order is considered as paid.

**Comment by MatShir** [comment]
https://github.com/PrestaShop/PrestaShop/pull/17852#issuecomment-603937620

**Pull request merged**